src/actionCreators/index.js

```diff
@@ -37,18 +38,29 @@ const apiPollTimeout = 3000;
 const maxStatusCheckCount = (120 * 1000) / apiPollTimeout;
 let statusCheckCount = 0;
 
-export const createSigningJob = files => dispatch => {
+const authenticationHeaderVaule = () => {
+  try {
+    const token = JSON.parse(localStorage.getItem('auth')).id_token.bearer;
+    return `Bearer ${token}`;
+  } catch (err) {
+    console.log(err.message);
```

Comment:
Possible errors that could happen are 'can't get id_token of undefined' or something similar. That or a json parsing error. Do we need these logged out?